core/domain/blog_services.py

```diff
@@ -413,19 +416,22 @@ def get_published_blog_post_summaries_by_user_id(
         list(BlogPostSummary). The summary objects associated with the
         blog posts assigned to given user.
     """
-    blog_rights_models = (
-        blog_models.BlogPostRightsModel.get_published_models_by_user(
-            user_id, offset, max_limit
+    blog_post_summary_models: Sequence[blog_models.BlogPostSummaryModel] = (
+        blog_models.BlogPostSummaryModel.query(
+            blog_models.BlogPostSummaryModel.author_id == user_id
+        ).filter(
+            blog_models.BlogPostSummaryModel.published_on != None # pylint: disable=singleton-comparison, inequality-with-none, line-too-long
```

Comment:
```suggestion
            blog_models.BlogPostSummaryModel.published_on != None  # pylint: disable=singleton-comparison, inequality-with-none
```
Is the linter throwing an error without `line-too-long` ?. because this line does not exceed the 80 words limit

Reply:
yes it does throw line too long error